mix.exs

```diff
@@ -67,10 +68,20 @@ defmodule Nimble.MixProject do
   defp aliases do
     [
       codebase: ["format --check-formatted", "credo", "sobelow --config"],
+      coverage: ["coveralls.html --raise"],
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
       "ecto.reset": ["ecto.drop", "ecto.setup"],
-      test: ["ecto.create --quiet", "ecto.migrate", "test"],
-      coverage: ["coveralls.html --raise"]
+      test: [
+        "ecto.create --quiet",
+        "ecto.migrate",
+        "assets.compile --quiet",
+        "test"
+      ],
+      "assets.compile": &compile_assets/1
     ]
   end
+
+  defp compile_assets(_) do
+    Mix.shell().cmd("cd assets && npm run build:dev", quiet: true)
```

Comment:
🤔 
```suggestion
    Mix.shell().cmd("npm run --prefix assets build:dev", quiet: true)
```

Reply:
Fixed it in https://github.com/nimblehq/elixir-templates/pull/17/commits/3ab0824fdbf8ff49c304cc194377a4728cbb964d